src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -91,21 +91,28 @@ class SideNav extends Component {
     }
 
     return (
-      <ul
+      <div
+        id="va-sidenav-wrapper"
         className={`usa-width-one-fourth va-sidenav vads-u-margin--1 medium-screen:vads-u-height--auto medium-screen:vads-u-margin-y--0 medium-screen:vads-u-margin-left--0 medium-screen:vads-u-margin-right--2p5 ${
           this.state.active ? `va-sidenav-height` : null
         }`}
       >
-        <span
+        <button
```

Comment:
NIT: Unless we want this to be a `submit` button, we should add `type="text"`. 

According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type)

> `submit`: The button submits the form data to the server. This is the default if the attribute is not specified for buttons associated with a <form>, or if the attribute is an empty or invalid value.

Reply:
Good catch, @billfienberg - will do :+1:

Reply:
![image](https://user-images.githubusercontent.com/12773166/81088807-c183fa00-8eb8-11ea-8401-4e05584ebb2c.png)

I think you meant `type="button"` 🙂

Reply:
Lol! Yeah, I just copy-pasted-didn't_think

Reply:
> I think you meant `type="button"` 🙂

🤦‍♂️. My bad. I did mean `type="button"`. Good catch.